packages/common/src/index.ts

```diff
@@ -407,6 +432,18 @@ export default class Common {
     return this._calcForkHash(hardfork)
   }
 
+  /**
+   *
+   * @param forkHash Fork hash as a hex string
+   * @returns Array with hardfork data (name, block, forkHash)
+   */
+  hardforkForForkHash(forkHash: string): any | null {
+    const resArray = this.hardforks().filter((hf: any) => {
+      return hf.forkHash === forkHash
+    })
+    return resArray.length === 1 ? resArray[0] : null
```

Comment:
I don't think we should support having two different hardforks with the same forkHash in the hardforks() list. Although this could happen since we "only" have 4 bytes of hash space.

Reply:
No, two different hardforks with the same forkHash shouldn't happen. This is more for catching the zero results case.